src/elements/stepper/stepper/stepper.ts

```diff
@@ -101,6 +101,7 @@ export class SbbStepperElement extends LitElement {
       this.querySelectorAll('form').forEach((form) => form.reset());
     }
     this.selectedIndex = 0;
+    this.selected?.label?.focus();
```

Comment:
Is this not a little bit invasive, as also, if programmatically reset, it steals the focus from anywhere to here?

Reply:
Not sure, If I don't do that the focus would remain on the reset button in the last step while it come back to the first one. So it made sense to me to focus on the first step label.

Reply:
what about just calling blur() on the reset button? Or otherwise, only call it if triggered by user action?